planter/planter.sh

```diff
@@ -27,4 +27,5 @@ REPO=${REPO:-${PWD}}
 VOLUMES="-v ${REPO}:${REPO} -v ${HOME}:${HOME} --tmpfs /tmp:exec,mode=777"
 GID="$(id -g ${USER})"
 ENV="-e USER=${USER} -e GID=${GID} -e UID=${UID} -e HOME=${HOME}"
-docker run --rm ${VOLUMES} --user ${UID} -w ${PWD} ${ENV} ${IMAGE} ${@}
+EXTRA="${DOCKER_EXTRA:-}"
```

Comment:
not sure you need this step - just use `${DOCKER_EXTRA:-}` directly on the next line.

Reply:
done, I think I had something even more needlessly fancy and then I got 80 character happy :-)